hw_7.py

```diff
@@ -0,0 +1,12 @@
+"""
+Write a program to count the number of lines in a text file.
+"""
+
+
+def count_rows():
+    with open('file/short documentation.txt', 'r') as f:
+        line = f.readlines()
```

Comment:
It's a lot of lines, not just a single line.

Reply:
done,renamed